modules/rstudio/hm.nix

```diff
@@ -0,0 +1,26 @@
+{
+  config,
+  lib,
+  ...
+}:
+{
+  options.stylix.targets.rstudio = {
+    enable = config.lib.stylix.mkEnableTarget "RStudio" false;
+  };
+
+  config =
+    lib.mkIf (config.stylix.enable && config.stylix.targets.rstudio.enable)
+      {
+        xdg.configFile."rstudio/themes/stylix.rstheme.generated" = {
+          source = config.lib.stylix.colors {
+            template = ./base.rstheme.mustache;
+            extension = ".rstheme.generated";
+          };
+          onChange = ''
+            rm -f ${config.xdg.configHome}/rstudio/themes/stylix.rstheme
+            cp ${config.xdg.configHome}/rstudio/themes/stylix.rstheme.generated ${config.xdg.configHome}/rstudio/themes/stylix.rstheme
+            chmod u+w ${config.xdg.configHome}/rstudio/themes/stylix.rstheme
+          '';
```

Comment:
The intention behind this is to `rm --force` manual changes outside Stylix, right? I think we should remove this, following Home Manager's principle of erroring about this.